ibis/backends/mysql/tests/conftest.py

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
That's an incorrect git guess, not really a rename, just one `__init__.py` removed, and one created. The `all` directory is now removed (moved to `backends/tests`), so that empty file is removed, and `mysql` had no `tests` directory, so I created it to add a `conftest.py` with its tests configuration, and also created that file.

A bit confusing, but you can ignore it.